mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -36,6 +40,13 @@ public void afterProjectsRead(MavenSession session) throws MavenExecutionExcepti
 
         configuration = Configuration.load();
 
+        if (session.getRequest().getLoggingLevel() == 0) {
+            enableMavenDebugLogLevel(true);
+            logger.debug("Maven build run in debug log level.");
+        }
+
+        logger.debug("Applied user properties: %s", session.getUserProperties());
```

Comment:
Same for this line - I don't really think that we need this. This is nothing related only to smart testing - it contains all properties in general. Instead of this, you could log only the "smart testing" properties specified by user

Reply:
This returns all the properties set by the user in the mvn request. I feel it is useful to have scm properties and other properties set in the request like `skipTests` etc. that are not included in "smart testing" properties but influence our extension.
Eg: `DEBUG: Smart-Testing - Applied user properties: {smart.testing.version=0.0.1-SNAPSHOT, surefire.exitTimeout=-1, surefire.timeout=0, smart.testing.mode=ordering, skipITs=true, smart.testing=affected, scm.range.tail=HEAD~, skipTests=false, scm.range.head=HEAD, smart.testing.debug=true}
`